app/controllers/books_controller.rb

```diff
@@ -63,6 +64,10 @@ def destroy
 
   private
 
+  def exist_book?
+    redirect_to books_path unless Book.find_by(id: params[:id])
```

Comment:
```suggestion
    redirect_to books_path unless Book.find(params[:id])
```

idでの検索なのでfindメソッドで良さそうです🙆‍♀️